ReadMe.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.com/AtamaZack/ah-frontend-dojo.svg?branch=chore-travisCI-setup-165273520)](https://travis-ci.com/AtamaZack/ah-frontend-dojo)
+
```

Comment:
Good Work @AtamaZack. Though I wonder whether we will be using the badge attached to your repo or the Andela repo.

Reply:
Nice work @AtamaZack but I also think you should look into changing the reference of the badge to point to the Andela repo.

Reply:
Thanks for the feedback, resolved.